packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -237,53 +235,73 @@ const _generateStyles = (placeholder: boolean, error: boolean) => {
     }
 
     const focusHoverStyling = {
-        borderColor: error ? tokens.color.red : tokens.color.blue,
-        borderWidth: tokens.border.width.thin,
-        paddingLeft: adjustedPaddingLeft,
-        paddingRight: adjustedPaddingRight,
+        outlineColor: error
+            ? semanticColor.action.outlined.destructive.hover.border
+            : semanticColor.action.outlined.progressive.hover.border,
+        // Outline sits inside the border (inset)
+        outlineOffset: -border.width.thin,
+        outlineStyle: "solid",
+        outlineWidth: border.width.thin,
     };
-    const activePressedStyling = {
-        background: error ? tokens.color.fadedRed : tokens.color.fadedBlue,
-        borderColor: error ? tokens.color.red : tokens.color.activeBlue,
-        borderWidth: tokens.border.width.thin,
-        paddingLeft: adjustedPaddingLeft,
-        paddingRight: adjustedPaddingRight,
+    const pressStyling = {
```

Comment:
@marcysutton I've updated the variable name to match the new convention: `press`.

Reply:
Oh nice, that looks consistent with the convention of state + "Styling"! The usage I was referencing in my prior comment is actually down at the bottom of the file, where `pressStyling` is used: `press: pressStyling`. The `press` part is what I was curious about, since it differs from `disabled` in how we reference state!

Reply:
Ahh I see! `press` is a new naming convention that we are using with Design. Wonder Blocks and Thunder Blocks are going to be updated soon to reflect that change as well to make it a bit more platform agnostic and match the other pseudo-states we currently define (`default/rest`, `hover`, `focus`). As for `disabled`, we are keeping it that way to keep it consistent with the `disabled`/`aria-disabled` HTML/ARIA attributes.